region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -403,19 +403,14 @@
     {
       "id": 3,
       "link": [1, 1],
-      "name": "Sciser Farm",
+      "name": "Sciser Farm (2 Scisers)",
       "requires": [
         {"resetRoom": {"nodes": [1]}},
-        {"partialRefill": {"type": "Energy", "limit": 240}},
-        {"partialRefill": {"type": "PowerBomb", "limit": 6}},
-        {"or": [
-          "Gravity",
-          "Spazer",
-          "Wave",
-          "Plasma",
-          "canUseGrapple",
-          "canBePatient"
-        ]}
+        {"cycleFrames": 860},
```

Comment:
Now that there are 2 crabs, at least grapple should be added as a farm weapon.  Jumping to the right loses a lot of time, which is why grapple is so strong.  And with Gravity you would use a different farm strat.